backend/db/migrate/20250810200038_add_unique_dividend_constraint.rb

```
@@ -0,0 +1,27 @@
+class AddUniqueDividendConstraint < ActiveRecord::Migration[8.0]
```

RuboCop (1.76.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)